system/java_certs.py

```diff
@@ -0,0 +1,234 @@
+#!/usr/bin/python
+#
+# (c) 2013, RSD Services S.A
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+# keytool -printcert -rfc -sslserver ppid.rsd-cloud.com:443 | keytool -importcert -keystore /usr/lib/jvm/java-7-oracle-amd64/jre/lib/security/cacerts -storepass changeit -alias ppid.rsd-cloud.com
+
+DOCUMENTATION = '''
+---
+module: java_cert
+version_added: "2.2"
+short_description: Uses keytool to import/remove key from java keystore(cacerts)
+description:
+  - This is a wrapper module around keytool. Which can be used to import/remove certificates from a given java keystore.
+options:
+  cert_url:
+    description:
+      - Basic URL to fetch SSL certificate from
+    required: true
+  cert_port:
+    description:
+      - Port to connect to URL. This will be used to create server URL:PORT
+    required: false
+    default: 443
+  cert_path:
+    description:
+      - Local path to load certificate from
+    required: false
+    default: null
+  cert_alias:
+    description:
+      - Imported certificate alias
+    required: false
+    default: null
+  keystore_path:
+    description:
+      - Path to keystore.
+    required: false
+    default: null
+  keystore_pass:
+    description:
+      - Keystore password
+    required: false
+    default: null
+  keystore_create:
+    description:
+      - Create keystore if it doesn't exist
+    required: false
+    default: null
+  state:
+    description:
+      - Defines action which can be either certificate import or removal.
+    choices: [ 'present', 'absent' ]
+    default: 'present'
```

Comment:
Hmm I checked other plugins and this seems that state actually has a default state in all of them.